lib/manageiq-providers-ovirt.rb

```diff
@@ -1,2 +1,10 @@
 require "manageiq/providers/ovirt/engine"
 require "manageiq/providers/ovirt/version"
+
+module ManageIQ
+  module Providers
+    module Redhat
+      autoload :Discovery, 'manageiq/providers/redhat/discovery'
```

Comment:
Do we need this?  The `app/models` directory is already autoloaded and it seems to work without this.